README.md

```diff
@@ -58,6 +58,7 @@ $ changelog -h
     -p, --patch            create a patch changelog
     -m, --minor            create a minor changelog
     -M, --major            create a major changelog
+    -t, --tag              generate from specific tag
```

Comment:
Once you've updated `cli.js`, can you re-run the help and put the updated output here?

Reply:
The updated help text:
```
$ bin/generate --help

  Usage: generate [options]

  Generate a changelog from git commits.


  Options:

    -V, --version          output the version number
    -p, --patch            create a patch changelog
    -m, --minor            create a minor changelog
    -M, --major            create a major changelog
    -t, --tag <range>      generate from specific tag or range (e.g. v1.2.3 or v1.2.3..v1.2.4)
    -x, --exclude <types>  exclude selected commit types (comma separated)
    -f, --file [file]      file to write to, defaults to ./CHANGELOG.md, use - for stdout
    -u, --repo-url [url]   specify the repo URL for commit links, defaults to checking the package.json
    -h, --help             output usage information
```

Does it look okay?

Reply:
Yup!